src/form/form-textarea.tsx

```diff
@@ -32,17 +32,6 @@ const FormTextareaComponent = (
         ...otherProps
     } = props;
 
-    const [stateValue, setStateValue] = useState<
```

Comment:
realised that the counter no longer works if using this as a uncontrolled component (check the behaviour of the stories with counter). hence the state is still needed to track the current input length

changes required:
1. handleChange calls `setStateValue()` and `onChange()`
2. `TextAreaCounter` should use `stateValue`
3. TextareaBase should use `value`
4. restore `useEffect` to sync `stateValue` with `value`

Reply:
there's still the problem of 4 not being synced if value is undergoes transformation